src/zope/i18n/format.py

```diff
@@ -467,8 +471,8 @@ def format(self, obj, pattern=None, rounding=True):
         if bin_pattern[PADDING2] is not None and pre_padding > 0:
             if bin_pattern[PADDING1] is not None:
                 text += bin_pattern[PADDING2]
-            else:
-                text += bin_pattern[PADDING2]*pre_padding
+            else: # pragma: no cover
+                text += bin_pattern[PADDING2] * pre_padding
```

Comment:
Too hard to test?

Reply:
Yes, sorry 🙁 It could be done, but there are a lot of branches and it's based on positions in a list and it was just too much to try to come up with a *meaningful* test...